Fish_Doku/Filter_Types/effects.md

```diff
@@ -1,12 +1,10 @@
-
-
 # Effect filters
 
 ## cue_list_filter
  - Type name: `cue_list_filter`
  - Type id: `44`
 - Input channels: 
-  * `time` of type of type `double`
+  * `time` of type of type `double`(default 0 with warning)
```

Comment:
I think we should not provide a default for this option as it is basically mandatory for this filter operation. We should either stick with an error message in case of not connected values (as it is now) or use the post-processing to include a time source filter. Last option would be actually reasonable as we're going to implement an optimization stage to eliminate redundant time sources anyway.

However we should definitely stick with the exposition of this node edge within the editor, as the user might wish to modify the time values provided to this filter.